internal/operations/operations.go

```diff
@@ -199,6 +199,47 @@ func Ping(ctx context.Context, dut binding.DUT, dest string, count int32) error
 	return nil
 }
 
+// GetSystemTime returns the current time in nanoseconds From epoch
+func GetSystemTime(ctx context.Context, dut binding.DUT) (uint64, error) {
+	gnoi, err := FetchGNOI(ctx, dut)
+	if err != nil {
+		return 0, err
+	}
+	resp, err := gnoi.System().Time(ctx, &spb.TimeRequest{})
+
+	if err != nil {
+		return 0, err
+	}
+	return resp.GetTime(), nil
+
+}
+
+// FactoryReset executes the Factory Reset RPC
```

Comment:
The comment should say something like "FactoryReset performs a factory reset of the device" and it should mention the purpose of the factoryOS and zeroFill arguments